### DISCUSSION TITLE
Show User ID in View Data

### DISCUSSION THREAD

**Discussion by shodan02** [arnoudkooi/SN-Utils#342] [Ideas]
Hello arnoudkooi!
Is it possible to somehow to display User ID in View Data table? primary_contact.user_name
I don't find any option for that right now.
Thanks!

**Comment by arnoudkooi** [comment]
Do you mean a dotwalked field? that is not supported, unfortunately...

**Reply by shodan02** [reply]
Yes i mean the uneditable field. If i go to the user page and view the data it is there. But if i check this in a CI page, i dont see it ...
![image](https://user-images.githubusercontent.com/119402002/208603609-933b26da-51d6-40b0-8c8b-74bf562c7099.png)

**Reply by arnoudkooi** [reply]
You probably see the name field, cause that is OOB the Display field for the user table
